docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -832,6 +839,10 @@ You can browse and download files in the |artifacts| section of the experiment.
 
     Keep an eye on your artifacts as they may consume a lot of storage. You can always remove some by using :meth:`~neptune.experiments.Experiment.delete_artifacts`.
 
+.. warning::
+
+    Make sure that you define correct path to files that you want to upload. If you pass directory then all its content is uploaded, which may result in unintended logging of large amount of data and cluttering your storage.
```

Comment:
Make sure that you define the correct path to files that you want to upload. If you pass the directory, then all its content is uploaded, resulting in unintended logging of a large amount of data and cluttering your storage.

Reply:
ok